sdk/core/azure-core/inc/azure/core/internal/tracing/service_tracing.hpp

```diff
@@ -4,7 +4,8 @@
 #include "azure/core/context.hpp"
 #include "azure/core/internal/client_options.hpp"
 #include "azure/core/internal/http/user_agent.hpp"
-#include "azure/core/tracing/tracing.hpp"
+#include "azure/core/internal/tracing/tracing_impl.hpp"
+#include <utility>
```

Comment:
Just to learn, what's requiring this header to be included?

Reply:
`std::static_pointer_cast<T>`

But it's not actually used.